dashboard/app/models/levels/applab.rb

```diff
@@ -74,7 +74,11 @@ def self.palette_categories
   end
 
   def self.maker_palette_categories
-    %w(Maker Circuit)
+    %w(Maker Circuit micro:bit)
```

Comment:
Could you add a comment describing the distinction between palette categories and APIs?

Reply:
Tangentially, is there a mapping between palette categories and APIs, or are they completely distinct? Wondering if there's a way to simplify the error handling logic that looks like this:

```
(starting_category == 'Circuit' && makerlab_property != 'circuitPlayground') ||
      (starting_category == 'micro:bit' && makerlab_property != 'microbit') ||
      (starting_category == 'Maker' && !maker_enabled)
```

Reply:
Oof, I definitely was wondering about this. The starting category is as displayed to the user and the makerlab_property is as saved to the main.json, so a bit different user case. I don't think we can align them perfectly, but we could probably make a look-up between the two to minimize potential error here.